pkgs/dart_model/test/model_test.dart

```diff
@@ -107,6 +107,34 @@ void main() {
           expected['uris']!['package:dart_model/dart_model.dart']!['scopes']![
               'JsonData']!['members']);
     });
+
+    test('can give the path to Members in buffer backed maps', () {
+      final member = model.uris['package:dart_model/dart_model.dart']!
+          .scopes['JsonData']!.members['_root']!;
+      expect(model.pathToMember(member),
+          '/uris/package:dart_model/dart_model.dart/scopes/JsonData/members/_root');
```

Comment:
This doesn't look like a very useful path, how are you supposed to parse out the library URI? What is the intended use case for these paths?

Reply:
Mentioned elsewhere :) indeed I expect it will change to path segments or maybe something even more specific like a lookup of exactly library URI.

It is intended for augmentation builders which we don't have quite yet :)